public/aboutus.html

```diff
@@ -0,0 +1,230 @@
+<p> </p>
+<p><strong>Who else is behind SG Kampung?           </strong></p>
+<p>SG Kampung is a GA Student Project by Roy Koh for a online resident focal point in the new BTO flat he is shifting to.</p>
+<p>SG Kampung is based in Singapore, Woodlands. It is looking towards building a team of people passionate about marrying technology with community to help build relationships between neighbours across Singapore.</p>
+<p><img class="wysiwyg-text-align-center" src="/hc/en-us/article_attachments/202822865/Screen_Shot_2016-07-10_at_2.47.48_pm.png" alt=""></p>
+<p><strong>Privacy </strong></p>
+<p>The privacy of our members is important to us. We want you to feel safe sharing information online that you would be happy to share over the fence with a neighbour.</p>
+<ul>
+<li>We ask that all members use their real name when creating their account.</li>
+<li>We ask that all members verify their address</li>
+<li>We do not share your personal information with any advertisers or third parties</li>
+<li>SG Kampung is protected by a password and encrypted by HTTPS</li>
+<li>All data is stored in Singapore and is protected under Singaporen privacy laws</li>
+</ul>
+<p> </p>
+  </div>
+
+  <div class="article-attachments">
+    <ul class="attachments">
+
+    </ul>
+  </div>
+
+<input id="hdnTitle" type="hidden" value="SG Kampung Help Centre & FAQ" />
+
+</article>
+
+<aside class="article-sidebar side-column">
+  <div data-recent-articles></div>
+
+  <section class="related-articles">
+    <h3>Related articles</h3>
+    <ul>
+
+        <li>
+          <a href="/hc/en-us/articles/204179795-What-are-SG Kampung-Constituent-Leads-">What are SG Kampung Constituent Leads?</a>
+        </li>
+
+        <li>
+          <a href="/hc/en-us/articles/202376905-Getting-to-know-SG Kampung">Getting to know SG Kampung</a>
+        </li>
+
+        <li>
+          <a href="/hc/en-us/articles/201654899-Terms-of-Use">Terms of Use</a>
+        </li>
+
+        <li>
+          <a href="/hc/en-us/articles/202376695-Getting-started-">Getting started </a>
+        </li>
+
+        <li>
+          <a href="/hc/en-us/articles/203764279-Creating-the-right-account-">Creating the right account </a>
+        </li>
+
+    </ul>
+  </section>
+
+
+</aside>
+  </main>
+
+  <footer class="footer">
+  <div class="footer-inner">
+    <!-- Footer content -->
+    <h6>SG Kampung Community 2016</h6>
+    <a href="../public/aboutus.html" target="_blank">About Us</a>
+    <a href="../public/organizations.html">Organisations</a>
+    <a href="../public/events.html">Events</a>
+    <a href="../public/blog.html" target="_blank">Blog</a>
+    <a href="../public/leads.html" target="_blank">Leads</a>
+    <a href="../public/privacypolicy.html" target="_blank">Privacy Policy</a>
+    <a href="../public/tou.html" target="_blank">Terms of Use</a>
+    <a href="../public/help.html" target="_blank">Help</a>
+    <a class="last" href="../public/contactus.html" target="_blank">Contact Us</a>
```

Comment:
Class `last` can be achieved with `:last-of-type` or `:last-child` more elegantly.